README.md

```diff
@@ -22,15 +22,16 @@ can find a preview of the new documentation at
 - [x] Implement broken links check
 - [x] Upgrade to Starlight 0.32 and re-implement Starlight Utils
 - [x] Make logo href be tenzir.com
-- [ ] Rename *Documentation* to *Docs* in header
-- [ ] Tweak `index.mdx` title
+- [x] Rename *Documentation* to *Docs* in header
+- [x] Tweak `index.mdx` title
 
 ### CI / Automation
 
 - [x] Update `tenzir.yaml.example` via CI
 - [x] Update OpenAPI spec via CI
 - [x] Update `tql.tmLanguage.json` via CI
-- [ ] Create consolidated Node & Platform changelog
+- [x] Create consolidated Node & Platform changelog
+  - [ ] Figure out how to use changesets
```

Comment:
I don't think we can, realistically. They're made for mono repos. However, we can very easily use the same approach with a few CI jobs and some shell scripts. It's far from rocket science.

Reply:
Why? Is it because of the UUID generation?

Reply:
Taking this offline, as it's not relevant to this PR.